### DISCUSSION TITLE
Tests for CLC solid phase thermo state block

### DISCUSSION THREAD

**Discussion by ravindurra** [IDAES/idaes-pse#283] [Q&A] [answered]
I'm trying to run the tests for CLC solid phase thermo state block. But there isn't any outcome. Can anyone explain what I'm doing wrong in here?  
Below is the original test in IDAES

`
def solid_prop():
    m = ConcreteModel()
    m.fs = FlowsheetBlock(default={"dynamic": False})
    m.fs.properties = SolidPhaseThermoParameterBlock()
    m.fs.unit = m.fs.properties.build_state_block(default={"parameters": m.fs.properties, "defined_state": True})
    m.fs.unit.flow_mass.fix(1)
    m.fs.unit.particle_porosity.fix(0.27)
    m.fs.unit.temperature.fix(1183.15)
    m.fs.unit.mass_frac_comp["Fe2O3"].fix(0.45)
    m.fs.unit.mass_frac_comp["Fe3O4"].fix(1e-9)
    m.fs.unit.mass_frac_comp["Al2O3"].fix(0.55)
    return m

def test_initialize(solid_prop):
    initialization_tester(
            solid_prop)
`

`para = solid_prop()
output = test_initialize(solid_prop=para)
output
`

**Comment by andrewlee94** [comment, marked as answer]
Hi @ravindurra

First of all, why are you writing this as a pytest file and not a normal Python script? Test scripts are generally only used for automated testing of code, and not actually running examples and examining the results. Normally, tests do not give any output unless something fails, so no output is actually the desired result in most cases. Thus, seeing as you are getting no output that generally means the tests are working correctly.

To actually run and example and get output, you generally need to write a Python script instead, and specifically display some output from the model.

**Comment by ksbeattie** [comment]
@ravindurra, given the time passed, I'm going to mark @andrewlee94's comment as an answer to your question.  Please re-open if that's not the case.
